integrations/http/client.go

```diff
@@ -10,62 +10,56 @@ import (
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
-var args = sdkmodule.WithArgs(
-	"url",
-	"params?",
-	"headers?",
-	"raw_body?",
-	"form_body?",
-	"json_body?",
-	// TODO: Mismatched naming, see http.go lines 57-59.
-	"form_encoding?",
-	"auth?",
-)
-
 var IntegrationID = sdktypes.IntegrationIDFromName("http")
 
 var desc = kittehs.Must1(sdktypes.StrictIntegrationFromProto(&sdktypes.IntegrationPB{
 	IntegrationId: IntegrationID.String(),
 	UniqueName:    "http",
-	DisplayName:   "HTTP",
+	DisplayName:   "HTTP / REST",
 	LogoUrl:       "/static/images/http.svg",
-	// TODO: Integration documentation link.
+	ConnectionUrl: "/httprest/connect",
```

Comment:
technically I agree, but:
1. Even though all REST APIs are HTTP APIs, a majority of people refer to them simply as REST APIs without mentioning HTTP
2. The real reason I added "rest" to the URL path was to avoid conflict with all `/http/` webhooks, e.g. in case someone creates a project called "connect"

Reply:
I'm fine with changing the integration to use `/webhook/...` instead of `/http/...` for webhooks, and then change the connection links to `/http/connect` and `/http/save` - but that's a behavior change that should be discussed in a larger forum. WDYT?

Reply:
regarding user projects, we have to have some prefix that split out endpoints from user endpoints so we can have what ever routes we need without worrying about project names users can have

regarding the renaming, I think we should do it, just need to sync with haim

good way to test this sync between docs and code

Reply:
this shouldn't be called rest. this is a generic http endpoint.

Reply:
Done:
- Removed all mentions of "REST" (in web pages and URL paths)
- HTTP new-connection web pages now use `/i/http/...` paths
- No impact on existing webhooks, which continue to use `/http/...`
- In the future, all integrations will be under `/i` - but this is beyond the scope of this specific PR

Reply:
https://linear.app/autokitteh/issue/ENG-677/improvements-in-integration-web-uis